lib/matplotlib/collections.py

```diff
@@ -1302,12 +1316,14 @@ def _add_offsets(self, segs):
 
     def set_color(self, c):
         """
-        Set the color(s) of the line collection.  *c* can be a
-        matplotlib color arg (all patches have same color), or a
-        sequence or rgba tuples; if it is a sequence the patches will
-        cycle through the sequence.
+        Set the color(s) of the line collection.
 
-        ACCEPTS: matplotlib color arg or sequence of rgba tuples
+        Parameters
+        ----------
+        c :
+            Matplotlib color argument (all patches have same color), or a
```

Comment:
Again, as for the *colors* parameter in the class constructor, it seems to me that one does not have anymore to use exclusively an RGBA sequence to define every color independently. See the second half of the example that I gave above.

NB: looking at the former docstring (more precisely to the “ACCEPTS:...” part), I have the feeling that there was a typo in “... a sequence ~~or~~ **of** rgba tuples; ...”.